frontend/src/components/Pages/FrontPage/FrontPage.tsx

```diff
@@ -15,6 +16,8 @@ const StyledFrontPage = styled.div`
 `
 
 export const FrontPage = () => {
+    redirectIfNoInstallationSelected()
```

Comment:
If we wanted to be even more optimal we could useMemo, or a useEffect with empty conditionals here, to prevent this function from needing to be rerun on each render, but it's not too important.